mrpt_graphslam_2d/CMakeLists.txt

```diff
@@ -18,3 +18,49 @@ if (BUILD_TESTING)
     ${test_output_directory}/${PROJECT_TEST_NAME})
 endif()
 
+set(PROJECT_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../)
+
+#V-REP Config
+################################################################################
+
+if (EXISTS "${PROJECT_ROOT_DIR}/../V-REP")
+  message(STATUS "Found V-REP")
+  set (V-REP_ROOT "${PROJECT_ROOT_DIR}/../" CACHE "V-REP root dir" STRING)
+
+  execute_process (COMMAND "/bin/bash" "-c" "cd ${V-REP_ROOT}; echo -n `pwd`" OUTPUT_VARIABLE V-REP_ROOT)
```

Comment:
If you want to get the absolute path for a given relative path, there are direct CMake commands to do it. 
Otherwise, I don't get what you're trying to do here... (?)

Also, invoking bash is not portable...

Reply:
I'll change this and use the direct cmake commands .